lib/container.js

```diff
@@ -165,7 +165,11 @@ function createHelpers(config) {
       } else {
         moduleName = `./helper/${helperName}`; // built-in helper
       }
-      const HelperClass = require(moduleName);
+
+      // @ts-ignore
+      // this handles the loading of custom helper. Either old syntax `export = HelperName` or new syntax `export default HelperName`
+      const HelperClass = moduleName.startsWith('./helper/') ? require(moduleName) : require(moduleName).default || require(moduleName);
```

Comment:
Could we split this into multiple operators as this becomes unreadable and it's hard to understand which operator is used in which context